### DISCUSSION TITLE
Limiting Player locaiton for their homes? `/cmi sethome homename 100000`

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#7284] [Enhancement]
Player on my server can have a maximum of 6 homes, yet this command `/cmi sethome homename 100000` allows them to put a home onto a page 90 pages away to try and hide a home from staff when staff use `/homes <username>`  Can we have a permission for this? Or at the very least limit the amount of pages they can make?

- Permission to use this number to set where it goes in the GUI
- Limit the amount of pages players can create via the homes.yml
- a `/homeslist <username>` command so staff can bypass the GUI if they want to

**Comment by Zrips** [comment]
Will add option to define max slot number player can define while creating homes
